sdcm/send_email.py

```diff
@@ -228,7 +233,7 @@ class LongevityEmailReporter(BaseEmailReporter):
               'username', 'nodes', 'events_summary']
 
     def cut_report_data(self, report_data, attachments_data, reason):
-        if ['test_status'] in attachments_data and len(attachments_data['test_status']) > 2 and len(
+        if attachments_data and 'test_status' in attachments_data and len(attachments_data['test_status']) > 2 and len(
```

Comment:
Why leave >2, ['test_status'] has length 2 always

Reply:
Thanks Alex, second time you point it out to me ]